apps/prairielearn/src/pages/instructorCourseAdminSharing/instructorCourseAdminSharing.html.ts

```diff
@@ -179,25 +179,27 @@ export function InstructorCourseAdminSharing({
                   <th>Sharing name</th>
                   <td data-testid="sharing-name">
                     ${sharingName !== null ? sharingName : ''}
-                    ${isCourseOwner
-                      ? html`
-                          <button
-                            type="button"
-                            class="btn btn-xs btn-secondary mx-2"
-                            id="chooseSharingName"
-                            title="Choose Sharing Name"
-                            data-toggle="modal"
-                            data-target="#chooseSharingNameModal"
-                          >
-                            <i class="fas fa-share-nodes" aria-hidden="true"></i>
-                            <span class="d-none d-sm-inline">Choose Sharing Name</span>
-                          </button>
-                          ${ChooseSharingNameModal({
-                            canChooseSharingName,
-                            csrfToken: resLocals.__csrf_token,
-                          })}
-                        `
-                      : ''}
+                    ${
```

Comment:
Hmm, why did this change? Ditto for the below change in this file.

Reply:
Had an unclosed `<h2>` that was causing formatting issues. Corrected in [c5187bb](https://github.com/PrairieLearn/PrairieLearn/pull/10483/commits/c5187bb79e07a5a99f75ef5cfa5a9961d6781e2d).